### PR TITLE
fix: handle MiniMax TTS timber weight configuration more robustly to avoid crashes on invalid or empty values

### LINKED ISSUE #8100: [Bug]新增MiniMax TTS API报错
### What happened / 发生了什么

报错信息：
实例化 minimax_tts_api(minimax_tts) 提供商适配器失败：Expecting value: line 1 column 1 (char 0)

### Reproduce / 如何复现？

新增MiniMax TTS API，点击保存

### AstrBot version, deployment method (e.g., Windows Docker Desktop deployment), provider used, and messaging platform used. / AstrBot 版本、部署方式（如 Windows Docker Desktop 部署）、使用的提供商、使用的消息平台适配器

Docker
AstrBot 4.24.2

### OS

Windows

### Logs / 报错日志

[2026-05-09 16:46:15.988] [Core]
[ERRO]
[v4.24.2] [provider.manager:711]: 实例化 minimax_tts_api(minimax_tts) 提供商适配器失败：Expecting value: line 1 column 1 (char 0)

### Are you willing to submit a PR? / 你愿意提交 PR 吗？

- [ ] Yes!

### Code of Conduct

- [x] I have read and agree to abide by the project's [Code of Conduct](https://docs.github.com/zh/site-policy/github-terms/github-community-code-of-conduct)。

### PR DESCRIPTION
**Pull request by Blueteemo**
### 问题描述
当用户保存 MiniMax TTS 配置时，`minimax-timber-weight` 字段可能为空字符串 `""`，此时 `json.loads("")` 会抛出 `JSONDecodeError`，导致 TTS 功能无法正常使用。

### 修复方案
1. 在解析 JSON 前增加空字符串判断
2. 增加 `try-except` 异常捕获，防止解析失败时程序崩溃
3. 解析失败时使用默认值作为兜底

### 修改内容
1. `minimax_tts_api_source.py`：提取 `default_timber_weight` 变量减少重复代码；添加 `logger.warning()` 记录 JSON 解析失败

### Related Issue
Fixes #8100

## Summary by Sourcery

Handle MiniMax TTS timber weight configuration more robustly to avoid crashes on invalid or empty values.

Bug Fixes:
- Prevent JSON parsing errors when `minimax-timber-weight` is an empty or invalid string by falling back to a default configuration.

Enhancements:
- Centralize the default timber weight configuration and log a warning when JSON parsing fails.

Tests:
- Add unit tests covering empty, whitespace, invalid, custom, and missing `minimax-timber-weight` configurations for MiniMax TTS.

### CODE REVIEW

**Comment by Blueteemo** [comment]
@sourcery-ai review

File: astrbot/core/core_lifecycle.py [hunk 1]
```diff
@@ -340,6 +340,12 @@ async def start(self) -> None:
         self._load()
         logger.info("AstrBot started.")
 
+        # Allow the event loop a chance to start the background tasks before emitting the signal.
+        # Emit the startup completion signal to stdout for AstrBot Launcher.
+        # The Launcher monitors stdout for this signal to determine when the instance is ready.
+        # This must use print (stdout) rather than logger (stderr) to be detectable by the Launcher.
+        await asyncio.sleep(0)
+        print("AstrBot 启动完成", flush=True)
```
Comment (Soulter) [inline astrbot/core/core_lifecycle.py]: 这是在干啥，和pr无关

**Review by Soulter — CHANGES_REQUESTED** [review]

**Review by Soulter — APPROVED** [review]
